keel/api/v1/cases/router.py

```diff
@@ -4,5 +4,5 @@
 urlpatterns = [
     path('create-cases', CaseView.as_view(), name="create-case"),
     path('list-cases', FilterUserCases.as_view(), name='list-cases'),
-    path('list-cases-details/<int:pk>', FilterUserCasesDetails.as_view(), name='list-cases-details'),
+    path('list-cases-details/<int:pk>', FilterUserCasesDetails.as_view({'get':'get_case'}), name='list-cases-details'),
```

Comment:
why would the name be list-cases-details :(

its only a detail of a single case